data/harisdev-netizen.json

```
@@ -41,4 +41,7 @@
       "icon": "FaLinkedin"
     }
   ]
+  "testimonials": [
+  "basit333",
```

Please remove the comma
```suggestion
  "basit333"
```